test/powershell/Modules/Microsoft.PowerShell.Security/ConstrainedLanguageRestriction.Tests.ps1

```diff
@@ -0,0 +1,917 @@
+
+        [Parameter()]
+        public SwitchParameter SetLockdownMode
+        {
+            get { return setLockdownMode; }
+            set { setLockdownMode = value; }
+        }
+        private SwitchParameter setLockdownMode;
+
+        [Parameter()]
+        public SwitchParameter RevertLockdownMode
+        {
+            get { return revertLockdownMode; }
+            set { revertLockdownMode = value; }
+        }
+        private SwitchParameter revertLockdownMode;
+        
+        protected override void BeginProcessing()
+        {
+            if (enableFullLanguageMode)
+            {
+                SessionState.LanguageMode = PSLanguageMode.FullLanguage;
+            }
+
+            if (enableConstrainedLanguageMode)
+            {
+                SessionState.LanguageMode = PSLanguageMode.ConstrainedLanguage;
+            }
+
+            if (setLockdownMode)
+            {
+                Environment.SetEnvironmentVariable("__PSLockdownPolicy", "0x80000007", EnvironmentVariableTarget.Machine);
+            }
+
+            if (revertLockdownMode)
+            {
+                Environment.SetEnvironmentVariable("__PSLockdownPolicy", null, EnvironmentVariableTarget.Machine);
+            }            
+        }
+    }
+'@
+
+    if (-not (Get-Command Invoke-LanguageModeTestingSupportCmdlet -ea Ignore))
+    {
+        $moduleName = Get-RandomFileName
+        $moduleDirectory = join-path $TestDrive\Modules $moduleName
+        if (-not (Test-Path $moduleDirectory))
+        {
+            $null = New-Item -ItemType Directory $moduleDirectory -Force
+        }
+
+        try 
+        { 
+            Add-Type -TypeDefinition $code -OutputAssembly $moduleDirectory\TestCmdletForConstrainedLanguage.dll -ErrorAction Ignore 
+        } catch {}
+
+        Import-Module -Name $moduleDirectory\TestCmdletForConstrainedLanguage.dll
+    }
+} # end if ($IsWindows)
+
+try
+{
+    $defaultParamValues = $PSDefaultParameterValues
+    $PSDefaultParameterValues["it:Skip"] = !$IsWindows
+
+    Describe "Built-ins work within constrained language" -Tags 'Feature','RequireAdminOnWindows' {
+
+        BeforeAll {
+            $TestCasesBuiltIn = @(
+                @{testName = "Verify built-in function"; scriptblock = { Get-Verb } }
+                @{testName = "Verify build-in error variable"; scriptblock = { Write-Error SomeError -ErrorVariable ErrorOutput -ErrorAction SilentlyContinue; $ErrorOutput } }
+            )
+        }
+
+        It "<testName>" -TestCases $TestCasesBuiltIn {
+
+            param ($scriptblock)
+
+            try
+            {
+                Invoke-LanguageModeTestingSupportCmdlet -EnableConstrainedLanguageMode
+
+                $result = (& $scriptblock)
+            }
+            finally
+            {
+                Invoke-LanguageModeTestingSupportCmdlet -EnableFullLanguageMode
+            }
+
+            ($result.Count -gt 0) | Should Be $true

```

Comment:
Use: Should BeGreaterThan

Gives better test failure message

Reply:
Changed.